### ISSUE TITLE
Default Toolbox powershell script uses $args as parametername for OnGameStarting, which is reserved

### ISSUE THREAD

**Issue by joyrider3774** [JosefNemec/Playnite#2068]
**Check for existing issue**
Did not see it reported

**Describe the bug**
Using toolbox to generate a new powershell script generates the following
```
function global:OnGameSelected()
{
    param(
        $args
    )    
}
```
Yet `$args.NewValue` or `$args.OldValue` can never be used probably because `$args` in powershell is already used for script arguments. if you rename `$args` to anything else for example `$arguments` or whatever it does work.

Might want to change the parameter name in that event that toolbox generates

**To Reproduce**
Make new powershell script and just add following line in ongameselected event. It will display an empty messagebox as it does not recognizes $args (i only added the showmessage line)

```
function global:OnGameSelected()
{
    param(
        $args
    )   
   $PlayniteApi.Dialogs.ShowMessage($args) 
}
```
Things like `$args.NewValue` and `$args.OldValue` won't work

If you change $args to for example $arguments it will start to work and correctly display the type GameSelectionEventArgs

```
function global:OnGameSelected()
{
	param(
		$arguments
	)
	$PlayniteApi.Dialogs.ShowMessage($arguments)
}
```
Things like `$arguments.NewValue` and `$arguments.OldValue` now also work

**Screenshots**

**Diagnostics ID**

**Comment by Bluscream** [comment]
Whats toolbox? Do you mean ISE?

**Comment by joyrider3774** [comment]
@Bluscream  https://playnite.link/docs/tutorials/toolbox.html

**Comment by JosefNemec** [comment]
Fixed in 8.4

**Comment by Bluscream** [comment]
> @Bluscream https://playnite.link/docs/tutorials/toolbox.html
> 
> like doing the following
> `Toolbox.exe new PowerShellScript test .` will create a test.ps1 powershell script with default events etc that contains the wrong named `$args` parameter in that event and which won't work

Now that's **useful**, thanks <3
